core/indexer/process/processTransactionsCommon.go

```diff
@@ -44,6 +44,12 @@ func addToAlteredAddresses(
 			TokenIdentifier: tx.EsdtTokenIdentifier,
```

Comment:
not necessarily needed, but I think that you can return here

Reply:
I will not remove it